app/src/main/java/com/github/siela1915/bootcamp/AutocompleteApi/IngredientAutocomplete.java

```diff
@@ -75,6 +90,7 @@ public void onResponse(Call<List<ApiResponse>> call, Response<List<ApiResponse>>
                             //Adds the ingredients to the passed list
                             ingredients.add(ing.name);
                             idMap.put(ing.name, ing.id);
+                            System.out.println(ing.name + ing.id);
```

Comment:
Are you meant to print to the console or was this left from the debugging phase?

Reply:
This was meant for debugging, I'll update it.